packages/dapp/src/components/PaymentDetailsForm.jsx

```diff
@@ -86,7 +86,7 @@ export const PaymentDetailsForm = ({ display, tokenData, allTokens }) => {
           setProviderInvalid(!utils.isAddress(v));
         }}
         error={providerInvalid ? 'Invalid Address' : ''}
-        tooltip="Recipient of the funds"
+        tooltip="This is your wallet address. It’s how you access this invoice & where you’ll receive funds released from escrow. It’s essential you have control of this address. (Do NOT use a multi-sig address)."
```

Comment:
It would be better to state first sentence: "This is the wallet address of the recipient/provider." I'd prefer to get way from using the you, your and you're terms. There could be times the client creates the smart invoice, so better to make it sound more neutral for those cases.